docs/network-spec/connection-manager.tex

```diff
@@ -950,11 +950,34 @@ \subsubsection{\Prune{} transitions}
 \connmngr{} will check if the server hard limit was exceeded. If that
 happened, the \connmngr{} will reset an arbitrary connection (with some preference).
 
-We prefer to reset an inbound connection rather than close an outbound
+There are actually two more transitions that can lead to exceeding server hard limit:
+\begin{itemize}
+  \item \OutboundStateDupTau{} to \InboundStateDup{} (via \DemotedToColdDupLoc{})
+  \item \OutboundStateDupTau{} to \DuplexState{} (via \PromotedToWarmDupRem{})
+\end{itemize}
+
+The reason why going from \OutboundStateDupTau{} to \InboundStateDup{} might exceed the
+server hard limit is exacty the same as the \DuplexState{} to \InboundStateDup{} one.
+However, the reason why going from \OutboundStateDupTau{} to \DuplexState{} might exceed
+the hard limit is more tricky. To reach a \DuplexState{} one assumes there must have been
+an incoming \textit{accepted} connection, but there's another way that two end-points
+can establish a connection without a node accepting it. If two nodes try to request an
+outbound connection simultaneously, it is possible, although improbable,
```

Comment:
I wouldn't say it's improbable.  Karl has been observing simultaneous open connections regularly.  It's all about timing, if the two server are far enough of each other and there's some event that triggers both of them to reconnect (a previous connection failed, and they re-connect after some small delay), then it becomes quite probable in a large network.